lib/iex/lib/iex/dochelp/erlang_stub.ex

```diff
@@ -0,0 +1,48 @@
+defmodule IEx.DocHelp.ErlangStub do
+  @moduledoc """
+  Stub module to suggest possible addons for Erlang documentation.
+
+  Currently only returns {:unknown, doc_list }
+  with a suggestive error message.
+  """
+
+  @suggestive "and currently there is no helper installed to provide Erlang documentation"
```

Comment:
But that's the point, you can install something to remedy this, but it will
be an external
platform specific module.



On Mon, Oct 12, 2015 at 12:33 PM, Matt <notifications@github.com> wrote:

> In lib/iex/lib/iex/dochelp/erlang_stub.ex
> <https://github.com/elixir-lang/elixir/pull/3640#discussion_r41792438>:
>
> > @@ -0,0 +1,48 @@
> > +defmodule IEx.DocHelp.ErlangStub do
> > +  @moduledoc """
> > +  Stub module to suggest possible addons for Erlang documentation.
> > +
> > +  Currently only returns {:unknown, doc_list }
> > +  with a suggestive error message.
> > +  """
> > +
> > +  @suggestive "and currently there is no helper installed to provide Erlang documentation"
>
> This message makes it appear as if you are missing something that needs to
> be installed (when I don't believe there is anything that can be installed
> to remedy this).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/3640/files#r41792438>.
>